src/index.html

```diff
@@ -20,8 +21,234 @@
       href="styles/index.scss"
     />
   </head>
-
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="Company Logo"
+          />
+        </a>
+      </div>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Huawey</a></li>
+          <li>
+            <a
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+          <li><a href="#">Accessories</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="Catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```

Comment:
The product title is repeated across multiple cards. Ensure this is intentional or update each card with unique product details.